### PR TITLE
fix: rename "Cost Center Wise Financial Statements" Report to "Cost Center Financial Statements" Report

### PR DESCRIPTION
**Pull request by hitenshKharva**
Ref : [Asana Task](https://app.asana.com/0/1199082161983365/1199966341817846)

image

### CODE REVIEW

File: erpnext/accounts/report/cost_center_financial_statements/cost_center_financial_statements.js [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+			"options": erpnext.get_presentation_currency_list(),
+			"default": frappe.defaults.get_user_default("Currency")
+		},
+		{
+			"fieldname": "cost_center",
+			"label": __("Cost Center"),
+			"fieldtype": "MultiSelectList",
+			get_data: function(txt) {
+				console.log(frappe.db.get_value('Company', frappe.query_report.get_filter_value("company"), "cost_center"));
+				console.log(frappe.query_report.get_filter_value("company"));
```
Comment (hrwX) [inline erpnext/accounts/report/cost_center_financial_statements/cost_center_financial_statements.js]:
```suggestion
```

File: erpnext/accounts/report/cost_center_financial_statements/cost_center_financial_statements.js [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+				return frappe.db.get_link_options('Cost Center', txt, {
+					company: frappe.query_report.get_filter_value("company")
+
```
Comment (hrwX) [inline erpnext/accounts/report/cost_center_financial_statements/cost_center_financial_statements.js]:
```suggestion
```

File: erpnext/patches.txt [hunk 3]
```diff
@@ -715,4 +715,5 @@ erpnext.patches.v13_0.delete_package_tag_property_setter_search_fields
 erpnext.patches.v13_0.set_batch_qty_in_batch
 erpnext.patches.v13_0.set_stock_valuation_method #2021-03-15
 erpnext.patches.v13_0.delete_bank_reconciliation_doctype
-erpnext.patches.v13_0.set_is_used #01-04-2021
\ No newline at end of file
+erpnext.patches.v13_0.set_is_used #01-04-2021
+execute:frappe.delete_doc("Report", "Cost Center Wise Financial Statements")
```
Comment (hrwX) [inline erpnext/patches.txt]:
```suggestion
execute:frappe.delete_doc_if_exists("Report", "Cost Center Wise Financial Statements")
```

**Review by hrwX — DISMISSED** [review]

**Review by rvpasari — DISMISSED** [review]

File: erpnext/accounts/report/cost_center_financial_statements/cost_center_financial_statements.js [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+			"fieldtype": "Select",
+			"options": erpnext.get_presentation_currency_list(),
+			"default": frappe.defaults.get_user_default("Currency")
+		},
+		{
+			"fieldname": "cost_center",
+			"label": __("Cost Center"),
+			"fieldtype": "MultiSelectList",
+			get_data: function(txt) {
+				return frappe.db.get_link_options('Cost Center', txt, {
```
Comment (rvpasari) [inline erpnext/accounts/report/cost_center_financial_statements/cost_center_financial_statements.js]: This is an interesting function, will discuss @hrwX

**Comment by sahil28297** [comment]
@hitenshKharva Did you delete the files for the existing "wise" report? I cannot see old files being deleted in this PR.

**Comment by hitenshKharva** [comment]
@sahil28297 I deleted files and tried to commit, Git shows files renamed. Therefore, I wrote a patch to delete the old files. I have committed deletion of files. Review it and let me know the changes.

**Review by sahil28297 — APPROVED** [review]

**Pull request merged**